RELEASE/scripts/autoscend.ash

```diff
@@ -1107,6 +1107,21 @@ boolean dailyEvents()
 		}
 	}
 
+	if(!isActuallyEd() && my_daycount() > 1)
```

Comment:
also check `canChew(...)` to make sure it can be consumed. Doesn't check spleen left but that is fine. For example this would make it so breath doesn't get pulled in BHY